sim/druid/lacerate.go

```diff
@@ -0,0 +1,121 @@
+package druid
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+)
+
+func (druid *Druid) registerLacerateSpell() {
+	if !druid.HasRune(proto.DruidRune_RuneLegsLacerate) {
+		return
+	}
+	initialDamageMul := 1.0
+
+	switch druid.Ranged().ID {
+	case IdolOfCruelty:
+		initialDamageMul += .07
+	}
+	rageMetrics := druid.NewRageMetrics(core.ActionID{SpellID: 431446})
+
+	druid.Lacerate = druid.RegisterSpell(Bear, core.SpellConfig{
+		ActionID:    core.ActionID{SpellID: 414644},
+		SpellSchool: core.SpellSchoolPhysical,
+		DefenseType: core.DefenseTypeMelee,
+		ProcMask:    core.ProcMaskMeleeMHSpecial,
+		Flags:       SpellFlagOmen | core.SpellFlagMeleeMetrics | core.SpellFlagAPL,
+
+		RageCost: core.RageCostOptions{
+			Cost:   core.TernaryFloat64(druid.BerserkAura.IsActive(), 0, 10),
```

Comment:
This only runs during initialization, so you'll have to dynamically modify the cost inside the aura's effects